### PR TITLE
initial plugin registry

### PR DESCRIPTION
**Pull request by NilsJacobsen**
Hey folks,

I was adding the plugin registry.
Since this is a subfolder in the source-code dir, I was assuming that this should also get changesets. 
To use this as dummy content to work on it on the visual side I created this first draft of the registry.

- Content is dummy content
- Schema should be final

**Schema:**
```ts
export type InlangPluginManifest = {
	id: string
	icon: string // url to an icon that can be embedded with <img src={plugin.icon}>
	keywords: string[] // searchable keywords
	repository: string // url, used to embed the readme
	contributorHandle: string // github handle of the contributor
	contributorImage: string // url to the contributor's image can be embedded with <img src={plugin.contributorImage}>
}
```

**Updated:**
```ts
export type InlangPluginManifest = {
	id: string
	icon: string // url to an icon that can be embedded with <img src={plugin.icon}>
	keywords: string[] // searchable keywords
	repository: string // url, used to embed the readme
}
```

### CODE REVIEW

**Comment by NilsJacobsen** [comment]
Okey guys wait i resolve the failing test before you make the review

File: source-code/plugin-registry/src/schema.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export type InlangPluginManifest = {
+	id: string
+	icon: string // url to an icon that can be embedded with <img src={plugin.icon}>
+	keywords: string[] // searchable keywords
+	repository: string // url, used to embed the readme
+	contributorHandle: string // github handle of the contributor
+	contributorImage: string // url to the contributor's image can be embedded with <img src={plugin.contributorImage}>
```
Comment (samuelstroschein) [inline source-code/plugin-registry/src/schema.ts:7]: Do we need contributor handle and image? Seems like repository url is enough?

File: source-code/plugin-registry/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+	"name": "@inlang/plugin-registry",
+	"type": "module",
+	"version": "0.0.1",
+	"private": true,
+	"exports": {
+		".": "./src/index.ts",
+		"./schema": "./src/schema.ts"
```
Comment (samuelstroschein) [inline source-code/plugin-registry/package.json:8]: 1. I would export the plugin schema as part of `index.ts` thus remove the `./schema` path. 
2. I would also move plugin list to a dedicated file named `plugins.ts` that is exported by `index.ts`. Everyone can open index.ts and immediately know what is being exported and what not.

File: source-code/plugin-registry/src/schema.ts (hunk 1 above)
Comment (NilsJacobsen) [reply source-code/plugin-registry/src/schema.ts:7]: I really like to give them back reputation, like raycast is doing it. But that is just my opinion. What is the rest thinking @inlang/team

Comment (samuelstroschein) [reply source-code/plugin-registry/src/schema.ts:7]: isn't the contributor contained in the repository url? 

https://github.com/samuelstroschein/inlang-plugin-json contains samuelstroschein. https://github.com/samuelstroschein.png is my avatar

Comment (NilsJacobsen) [reply source-code/plugin-registry/src/schema.ts:7]: Jap your right as long as you are not in an organisation

Comment (samuelstroschein) [reply source-code/plugin-registry/src/schema.ts:7]: If the contributor is an organization, the organization logo should be shown no? If so, just the repo url seems enough

File: source-code/plugin-registry/package.json (hunk 2 above)
Comment (NilsJacobsen) [reply source-code/plugin-registry/package.json]: Should be resolved

File: source-code/plugin-registry/src/schema.ts (hunk 1 above)
Comment (NilsJacobsen) [reply source-code/plugin-registry/src/schema.ts]: Convinced -> removed it

**Review by samuelstroschein — APPROVED** [review]

**Pull request merged**